### PR TITLE
SAPHanaSRTools.pm: align code with scale-out

### PR DESCRIPTION
**Pull request by fmherschel**
This PR is to bring the code of scale-out and scale-up more close together for the perl library SAPHanaSRTools.pm.

### CODE REVIEW

File: test/SAPHanaSRTools.pm [hunk 1]
```diff
@@ -1,12 +1,12 @@
 #
 #
 # SAPHanaSRTools.pm
-# (c) 2014 SUSE Linux Products GmbH
-# (c) 2015-2016 SUSE Linux GmbH
-# (c) 2017-2022 SUSE LLC
+# Copyright:    (c) 2014 SUSE Linux Products GmbH
+# Copyright:    (c) 2015-2016 SUSE Linux GmbH
+# Copyright:    (c) 2017-2020 SUSE LLC
```
Comment (angelabriel) [inline test/SAPHanaSRTools.pm:6]: We are going back in time for the Copyright :-)
Can you please fix this.

**Review by angelabriel — CHANGES_REQUESTED** [review]
Except the Copyright entry everything looks pretty fine.
Can you please fix the date.

File: test/SAPHanaSRTools.pm (hunk 1 above)
Comment (fmherschel) [reply test/SAPHanaSRTools.pm:6]: Good that you checked that. I will send an update asap.

Comment (angelabriel) [reply test/SAPHanaSRTools.pm]: Thanks

**Review by angelabriel — APPROVED** [review]
Thanks for the changes. Looks good to me. Thanks.

**Pull request merged**